packages/generator/generators/react-cmf/templates/package.json

```diff
@@ -41,19 +41,21 @@
     "eslint-plugin-import": "^1.16.0",
     "eslint-plugin-jsx-a11y": "^2.2.2",
     "eslint-plugin-react": "^6.3.0",
-    "extract-text-webpack-plugin": "^1.0.1",
-    "html-webpack-plugin": "2.22.0",
+    "extract-text-webpack-plugin": "^3.0.0",
+    "file-loader": "^0.11.1",
+    "html-webpack-plugin": "2.28.0",
     "jest-cli": "20.0.3",
     "node-sass": "4.5.3",
     "postcss-loader": "1.3.1",
     "react-addons-test-utils": "15.5.1",
     "react-test-renderer": "15.5.4",
-    "sass-loader": "4.1.1",
-    "style-loader": "0.16.1",
+    "resolve-url-loader": "^2.1.0",
+    "sass-loader": "6.0.5",
+    "style-loader": "0.18.2",
     "url-loader": "0.5.8",
-    "webpack": "^1.14.0",
-    "webpack-dashboard": "0.4.0",
-    "webpack-dev-server": "^1.16.5"
+    "webpack": "^3.6.0",
+    "webpack-dashboard": "^1.0.0-5",
+    "webpack-dev-server": "^2.5.0"
```

Comment:
webpack 3 is not supported accros the stack at the moment (not everywhere)
please align using version script provided in UI.
I think we should make the script update this package.json along the others